src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AutomaticEndAppealForNonPaymentEaHuTrigger.java

```diff
@@ -30,33 +31,68 @@ public class AutomaticEndAppealForNonPaymentEaHuTrigger implements PostSubmitCal
     int schedule14DaysInMinutes;
 
     public AutomaticEndAppealForNonPaymentEaHuTrigger(
-        DateProvider dateProvider,
-        Scheduler scheduler
+            DateProvider dateProvider,
+            Scheduler scheduler
     ) {
         this.dateProvider = dateProvider;
         this.scheduler = scheduler;
     }
 
     public boolean canHandle(
-        Callback<AsylumCase> callback
+            Callback<AsylumCase> callback
     ) {
         requireNonNull(callback, "callback must not be null");
 
         AsylumCase asylumCase =
-            callback
-                .getCaseDetails()
-                .getCaseData();
+                callback
+                        .getCaseDetails()
+                        .getCaseData();
 
         Optional<RemissionType> remissionType = asylumCase.read(REMISSION_TYPE, RemissionType.class);
         Optional<AppealType> appealType = asylumCase.read(APPEAL_TYPE, AppealType.class);
 
         return callback.getEvent() == Event.SUBMIT_APPEAL
-               && (remissionType.isPresent() && remissionType.get() == RemissionType.NO_REMISSION)
-               && (appealType.isPresent() && (appealType.get() == AppealType.EA || appealType.get() == AppealType.HU));
+                && (remissionType.isPresent() && remissionType.get() == RemissionType.NO_REMISSION)
+                && (appealType.isPresent() && (appealType.get() == AppealType.EA || appealType.get() == AppealType.HU
+                || checkCaseIsEligibleForAutomaticEndAppeal(asylumCase)));
+    }
```

Comment:
Did you test the scenario for ADA and Appeal type EA. Does it still end?

Reply:
ADA + AppealType.EA does not end appeal. This is because remission type has no value. In order for any appeal (regardless of case data) to be eligible for automatic ending after 14 days, remissionType must = noRemission. This has been confirmed by Elli.